### ISSUE TITLE
Split on a range, merge and remove

### ISSUE THREAD

**Issue by bcamp1973** [coherentgraphics/cpdf-binaries#4]
I have a common pattern that I need to handle, but I think I'm making it more complicated than it needs to be (I'm not very comfortable in the terminal). The requirements are:

1. Split book into pages, each as ```p%%%.pdf```
2. Some single pages span multiple pages in the book, so they need to stay merged. For example: *Pages 36-37 are a single page that need to stay merged as ```p036.pdf```*
3. Blank pages should be removed

My first thought was to use 4 separate commands:

1. Remove p38 (blank): ```cpdf source.pdf 1-37,39-end -o source.pdf```
2. Split pages 1-35: ```cpdf source.pdf 1-35 AND -split -o p-%%%.pdf``
3. Pull out p36: ```cpdf source.pdf 36-37 -o p-036.pdf```
4. Split pages 38-end: ```cpdf source.pdf 38-end AND -split -label-startval 37 -o p-%%%.pdf```

Long story short...lots of typing and not getting the results I want :(

**Comment by johnwhitington** [comment]
Could you upload an example PDF file somewhere, so I can play with your example command lines?

**Comment by bcamp1973** [comment]
Sure, [here's a file](https://www.dropbox.com/s/85pdi1tql4h2yxv/source.pdf?dl=0) i've been playing with. Thx for looking into it!

**Comment by johnwhitington** [comment]
Your mistake here is that -label-startval is to do with page labels, and nothing to do with the names of split files. So your second set of split files are overwriting the first set.

If you're intending to buy a commercial license for cpdf (it looks like you're doing some commercial work), please email me directly using the contact address on the website, and we can consider adding a feature which does what you are trying to do with -label-startval 37 in your example.

**Comment by bcamp1973** [comment]
Thanks for looking into it @johnwhitington, in the meantime I've composed some scripts for Automator (Mac OS) that are meeting my needs. I'll see how these work for a bit and reconsider cpdf in the future if my needs change.
